src/m365/cli/commands/cli-issue.ts

```diff
@@ -0,0 +1,69 @@
+import { Logger } from '../../../cli';
+import Command, {
+  CommandOption
+} from '../../../Command';
+import GlobalOptions from '../../../GlobalOptions';
+import commands from '../commands';
+
+interface CommandArgs {
+  options: Options;
+}
+
+interface Options extends GlobalOptions {
+  type: string;
+}
+
+class CliIssueCommand extends Command {
+  public get name(): string {
+    return commands.ISSUE;
+  }
+
+  public get description(): string {
+    return 'Returns, or opens a URL that takes the user to the right place in the CLI GitHub repo to create a new issue reporting bug, feedback, ideas, etc.';
+  }
+
+  public getTelemetryProperties(args: CommandArgs): any {
+    const telemetryProps: any = super.getTelemetryProperties(args);
+    telemetryProps.type = args.options.type;
+    return telemetryProps;
+  }
+
+  public commandAction(logger: Logger, args: CommandArgs, cb: (err?: any) => void): void {
+    let issueLink: string = "";
+
+    switch (encodeURIComponent(args.options.type.toLocaleLowerCase())) {
+      case 'bug':
+        issueLink = 'https://aka.ms/cli-m365/bug';
+        break;
+      case 'command':
+        issueLink = 'https://aka.ms/cli-m365/new-command';
+        break;
+      case 'sample':
+        issueLink = 'https://aka.ms/cli-m365/new-sample-script';
+        break;
+    }
+
+    try {
+      logger.log(issueLink);
+      global.open(issueLink);
+      cb();
+    }
+    catch (err) {
+      return cb(`Could not open url ` + issueLink);
+    }
+  }
+
+  public options(): CommandOption[] {
+    const options: CommandOption[] = [
+      {
+        option: '-t, --type <type>',
+        autocomplete: ['Bug', 'Command', 'Sample']
+      }
+    ];
+
+    const parentOptions: CommandOption[] = super.options();
+    return options.concat(parentOptions);
+  }
+}
```

Comment:
Hey @appieschot, could we add validation to ensure that the specified issue type is valid?

Reply:
Yes, will implement asap and update. Thanks!